tensornetwork/backends/backend_factory.py

```diff
@@ -16,25 +16,30 @@
 from __future__ import division
 from __future__ import print_function
 
-from tensornetwork.backends.tensorflow import tensorflow_backend
-from tensornetwork.backends.numpy import numpy_backend
-from tensornetwork.backends.jax import jax_backend
-from tensornetwork.backends.shell import shell_backend
-from tensornetwork.backends.pytorch import pytorch_backend
-import tensornetwork.config as config_file
-_BACKENDS = {
-    "tensorflow": tensorflow_backend.TensorFlowBackend,
-    "numpy": numpy_backend.NumPyBackend,
-    "jax": jax_backend.JaxBackend,
-    "shell": shell_backend.ShellBackend,
-    "pytorch": pytorch_backend.PyTorchBackend
-}
+import importlib
 
 
 def get_backend(name, dtype):
+
+  _BACKENDS = {
```

Comment:
We can't do this, as it breaks imports internally. (We'll talk about this in person).

Reply:
Actually hold on let met see if that is actually true.

Reply:
Confirmed this is broken internally.